css.webpack.js

```diff
@@ -57,48 +35,29 @@ module.exports = (env) => {
           ]
         },
         {
+          // this inlines resources if they are less than 8KB
           test: /\.(png|jpe?g|gif|svg|eot|ttf|woff|woff2)$/i,
-          loader: 'url-loader',
-          options: {
-            limit: 8192,
-          },
+          type: 'asset'
         }
       ]
     },
     plugins: [
       // extracts CSS to a separate file
       new MiniCssExtractPlugin({
-        filename: env.development ? '../stylesheets/[name].css' : '../stylesheets/[name].[hash].css',
+        filename: env.development ? '../stylesheets/[name].css' : '../stylesheets/[name].[contenthash].css',
```

Comment:
Looks like the supported substitutions are `hash`, `contenthash` and `chunkhash`. The reason I went with `contenthash` was because the [webpack 5 migration guide recommended it](https://webpack.js.org/migrate/5/#clean-up-configuration)